feat(i18n): translate 03-chapter to Spanish (#629)
todogroup/ospology: ospo-book/content/es/03-chapter.md

### Una Introducción a los Maturity Models

El compromiso de una organización con el open source generalmente se sitúa a lo largo de una escala que va de lo táctico a lo estratégico. Los maturity models te ayudan a comprender en qué parte de esta escala se encuentran las diferentes partes de la organización y a tener conversaciones sobre si está en el lugar correcto.

Existen muchos maturity models de open source diferentes. Algunos son generales, otros especializados. Hay maturity models para gobiernos, ONGs, empresas y más, con versiones y subversiones para adaptarse a cualquier organización.

> NOTA: Los maturity models pueden verse como una prescripción de cómo deben desarrollarse las OSPOs o el compromiso con el open source. Puede ser tentador pensar que siempre es mejor aumentar la madurez. Pero, recuerda que debes considerar qué *nivel* de madurez es apropiado para tu OSPO, o cada función de tu OSPO. No todas las partes necesitan estar altamente desarrolladas. Es posible que ya entreguen el valor necesario sin mayor desarrollo. Si los maturity models no se ajustan a tu OSPO, considera usar un modelo de capacidad o algo más que prefieras.

### Ejemplos de Maturity Models

> Cada uno de estos Maturity Models es ligeramente diferente, pero todos clasifican el compromiso con el open source desde lo táctico y menos intencional, hasta lo estratégico y más intencional.

Reviewer: ```suggestion
Cada uno de estos maturity models es ligeramente diferente, pero todos clasifican el compromiso con el open source desde lo táctico y menos intencional, hasta lo estratégico y más intencional.
```

*   Etapa 1: Proporcionar Cumplimiento de OSS, Inventario y Educación para Desarrolladores
*   Etapa 2: Evangelizar el Uso de OSS y la Participación en el Ecosistema
*   Etapa 3: Alojar Proyectos OSS y Hacer Crecer Comunidades
*   Etapa 4: Convertirse en un Socio Estratégico para la Toma de Decisiones

![OSPO Maturity Model](/images/ospo-maturity-model.jpg)

## Aplicando Esto a tu Organización

Aquí hay algunas sugerencias sobre cómo podrías usar las ideas y consejos anteriores para configurar tu OSPO. Estas se basan en el Modelo de Madurez 3 - The OSPO Maturity Model by The TODO Group.

### Usando una Checklist Simple

La *TODO OSPO checklist* [^8] ofrece un conjunto simplificado de hitos comunes tanto para OSPOs en etapa temprana como para las experimentadas, para navegar cada etapa del OSPO Maturity Model mencionado anteriormente. Ten en cuenta que una OSPO podría eliminar, agregar o editar parte del contenido de esta checklist para adaptarla a las necesidades de su organización.

### Usando Modelos de Madurez

Una vez que tengas cierta familiaridad con los modelos de madurez de open source, puedes comenzar a usar uno para construir tu estrategia y crear tu plan.

> El OSPO Japan Local Meetup Working Group, con el apoyo de The TODO Group y OpenChain, ha estado desarrollando una guía simple de preguntas frecuentes (FAQ) sobre OSPOs. Esta guía tiene como objetivo responder preguntas en cada paso del OSPO Maturity Model, que categoriza diferentes actividades de open source desde la etapa 0 a la 4, y describe el papel de la OSPO en cada nivel.

Reviewer: ```suggestion
El OSPO Japan Local Meetup Working Group, con el apoyo de The TODO Group y OpenChain, ha estado desarrollando una guía simple de preguntas frecuentes (FAQ) sobre OSPOs. Esta guía tiene como objetivo responder preguntas en cada paso del modelo de madurez de la OSPO, que categoriza diferentes actividades de open source desde la etapa 0 a la 4, y describe el papel de la OSPO en cada nivel.
```
Why are they capitalized and not translated?

Author: Yes, sorry, to be consistent, it is better if we take the term Maturity Model in every appearance and not translated.

Reviewer 2: I agree with @bringani .
Better to keep it as OSPO Maturity Model

Aquí hay algunos aspectos destacados de su trabajo para inspirarte:

![ospo-role](/images/ospo-role.png)
![benefits-of-oss](/images/benefits-of-oss.png)

> NOTA: Puedes encontrar un resumen de su trabajo tanto en japonés como en inglés en un artículo de Qiita escrito por uno de sus miembros [^9].

Mientras planificas la OSPO, es muy útil tener conversaciones 1:1 con gerentes, ejecutivos de alto nivel y trabajadores/contratistas de diferentes equipos que usan open source en sus operaciones diarias, o cuya estrategia implica tratar con proyectos open source (en términos de licencias, vulnerabilidades de seguridad). Utiliza los conocimientos de estas conversaciones para definir los motivadores únicos de la organización y mapearlos a áreas dentro de la organización donde el open source aporta valor.

Esto también ayudará a construir apoyo para tu trabajo en toda la empresa incluso antes de que la OSPO se cree y lance oficialmente.

> Mapea estos motivadores con diferentes tipos de actividad en toda la organización, utilizando el OSPO Maturity Model y creando una segunda división que categorice cada uno de estos motivadores únicos según las diferentes etapas. Usa esto como referencia cuando te relaciones y comuniques con tus stakeholders.

Reviewer: ```suggestion
Mapea estos motivadores con diferentes tipos de actividad en toda la organización, utilizando el modelo de madurez de la OSPO y creando una segunda división que categorice cada uno de estos motivadores únicos según las diferentes etapas. Usa esto como referencia cuando te relaciones y comuniques con tus stakeholders.
```

Why are they capitalized and not translated?

- The Evolution of the OSPO https://linuxfoundation.org/tools/the-evolution-of-the-open-source-program-office-ospo/
- OSPO 101 training module - OSPO and your organization: https://github.com/todogroup/ospo-career-path/tree/main/OSPO-101/module3

> ### Notas al Pie

Reviewer: Footnotes titles were translated in [chapter 2](eyJhbGciOiJSUzI1NiIsImtpZCI6IjRmY2ZiZjEyLWY2ODUtNDI5NS1iOTIwLWFhMWNkMTUyYzI0NCIsInR5cCI6IkpXVCJ9.eyJhZGRpdGlvbmFsX2luZm8iOnsiZW1haWwiOiJlc3RlYmFuLmRlYm9uaXN2aWFyQG1lcmNhZG9saWJyZS5jb20iLCJlbnZpcm9ubWVudCI6Im1lbGlfcHJvZCIsImZ1bGxfbmFtZSI6IkVzdGViYW4gTWlndWVsIERlIEJvbmlzIFZpYXIiLCJ0eXBlIjoidXNlciIsInVzZXJuYW1lIjoiZWRlYm9uaXMifSwiZXhwIjoxNzQ4MjkzNDY4LCJpYXQiOjE3NDgyNjEwNjgsImlkZW50aXR5IjoibXJuOnNlZ2luZjphZDp1c2VyL2VkZWJvbmlzIiwiaXNzIjoiZnVyeV90aWdlciIsInN1YiI6ImVkZWJvbmlzIn0.mRF5-COYXTtcTd7yUDJtElwuZqOVhQSh3VIFSrNIafBANp6oiQYExKFMrbOft4f5my5f7rjU4ZB5cX24-CvcS5B22BYQs8XpN5l55qV9UmHVI20oAudtvdgKcB-lv0loD2epLj78VlCjOQ-pSnyfz-8RW0rb2-tzeZ9axUm7IVg-AdgnrKsh_Sbp8bkYXjdb9By31lGH__Dp13smmPFmIo_f3v0x7c4fa-NhEE02bdgAGj7ZLuDe7LCi1xNt_8P8CTjf_hpwVV5y1Yj4TGzxDNbG-CTmlcWPzX5VHFFdMsiA26-cgX-i7sdkjE5NmPwnF7LNUqJU2YgZoVEZx_xD5Q)

It would be better to translate them to be consistent.
